### PR TITLE
Chat UI fixes

### PR DESCRIPTION
**Pull request by philipperolet**
- Sparkle components for clipboard, feedback
- proper fix of clipboard to avoid it overlaying text (float right)
- note: for clipboard, at the time used existing sparkle svg (ClipboardIconStroke, ClipboardIconSolid) rather than former  heroicons (ClipboardDocmentListIcon, ClipboardDocumentCheckIcon), cc @Duncid does it look good enough or would it be better with previous icons? in which case can you add them to sparkle :pray: ?
- note 2: linting error in tailwind.config.css => had it slipped through? or was there a reason?

### CODE REVIEW

**Comment by Duncid** [comment]
Can you post a screenshot?

File: front/components/use/chat/MessageFeedback.tsx [hunk 1]
```diff
@@ -19,40 +24,23 @@ export function MessageFeedback({
...
     <div
-      className={classNames("flex-end flex h-2 flex-row-reverse text-gray-400")}
+      className={classNames(
+        "flex-end flex h-2 flex-row-reverse pt-2 text-gray-400",
+        !message.feedback && hover ? "invisible group-hover:visible" : ""
+      )}
     >
-      <div
+      <IconButton
+        type={message.feedback === "positive" ? "primary" : "secondary"}
```
Comment (spolu) [inline front/components/use/chat/MessageFeedback.tsx:33]: Should be tertiary

File: front/components/use/chat/MessageFeedback.tsx [hunk 2]
```diff
@@ -19,40 +24,23 @@ export function MessageFeedback({
...
-          <HandThumbUpSolidIcon className="h-4 w-4" />
-        ) : (
-          <HandThumbUpSolidIcon className="h-4 w-4" />
-        )}
-      </div>
-      <div
+        className="ml-1"
+      />
+      <IconButton
+        type={message.feedback === "negative" ? "primary" : "secondary"}
```
Comment (spolu) [inline front/components/use/chat/MessageFeedback.tsx:39]: Should be tertiary (grey instead of black)

**Review by spolu — CHANGES_REQUESTED** [review]
Looks much nicer!! \o/

Minor change request.

**Comment by spolu** [comment]
THank for fixing the tailwind config. Looks very reasonable

**Comment by spolu** [comment]
@Duncid deploying on front-edge for you

**Comment by spolu** [comment]
Front edge is out @Duncid

**Comment by Duncid** [comment]
That's what I see:

image

We need to use Sparkle component:
- **IconButton** for clipboard
- **IconToggleButton** for up down thumbs
- In both case **Type = "tertiary"**

Use sparkle icons "hand-thumb-up", "hand-thumb-down", "clipboard" in the MINI version

**Comment by spolu** [comment]
Note mini version is -Icon (no solid no Stroke)

**Comment by philipperolet** [comment]
@Duncid thx
I changed to the mini icons & switched the clipboard to an IconButton rather than IconToggle. **To note**:
- icons in mini look the same as in solid / stroke in this situation
   - did you mean for better code quality we should use them rather than stroke / solid? if yes, good it's done :)
   - if you meant that you need the icons to look smaller, then I'll investigate (this is strange because height/width was not set anywhere else)
- I used a checkcircleicon for 1s as feedback to the user that the copy has been done

**IconButton vs IconToggleButton for thumbs**
I had originally used an IconToggle for the thumbs but it didn't work because toggling one should untoggle the other, and the current IconToggle component does not allow this behaviour AFAICT.  This is why I switched to IconButton

I can change the component to allow this behaviour and then use it if you think it's best @spolu @Duncid

**Review by spolu — DISMISSED** [review]
LGTM code wise 👍

**Pull request merged**